sample_trial/definition.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "sampletrial",
-    "id": "sample1",
+    "name": "mytrial",
```

Comment:
is this change intentional?

Reply:
yes. since others worked on this repo I didnt want to override their / mine deployment

Reply:
I think we should have a task to check that the folder name is the same as the name of the trial. Otherwise, the result can be unexpected...